src/main/java/com/meraphorce/controllers/UserController.java

```diff
@@ -16,12 +25,120 @@ public class UserController {
     private UserService userService;
 
     @PostMapping
-    public ResponseEntity<User> createUser(@RequestBody User user){
-        return ResponseEntity.ok(userService.createUser(user));
+    public ResponseEntity<ResponseDTO<?>> createUser(@RequestBody UserDTO user) {
+    	ResponseEntity<ResponseDTO<?>> response = null;
+    	ResponseDTO<?> responseDTO = null;
+    	UserDTO newUserData = null;
+    	
+    	newUserData = this.userService.createUser(user);
+    	
+    	if(newUserData != null) {
+    		responseDTO = ResponseDTO.builder()
+    				.object(newUserData)
```

Comment:
Why are you considering using another Layer of DTO ResponseDTO? Instead you could simply use the ResponseEntity from spring boot, that object returns the status and the body. Explain me your thoughts

Reply:
Yeah right! That's not make sense. I fix it out.